core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -88,7 +88,7 @@ UNWIND(	.fnstart)
 	 * PCR
 	 * - no change latency, enable clk gating
 	 */
-	mov_imm r0, 0x00004000
+	mov_imm r0, 0x00004800
```

Comment:
Indeed. The problem with using `SCTLR_RR` and `SCTLR_Z` in `generic_entry_a32.S` is that strictly speaking, those bits are ARMv7 only (and even not all implementations of ARMv7 support them although they are "don't care" when not supported), while `generic_entry_a32.S` is used for ARMv7 but also for ARMv8 AArch32 (although in this case, too, the bits are "don't care").
So, either we set `SCTLR_RR` and `SCTLR_Z` in `generic_entry_a32.S` (always? or subject to some `CFG_`? or using a platform define such as `plat-stm`'s `CPU_SCTLR_INIT`?)
Or we fix `plat-imx/a9_plat_init.S` so that is does `read_sctlr` + `write_sctlr`.